solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/issue_pass.rs

```diff
@@ -27,15 +62,23 @@ pub struct IssuePass<'info> {
         init,
         payer = payer,
         space = Pass::ON_CHAIN_SIZE,
-        seeds = [PASS_SEED, subject.as_ref(), network.key().as_ref(), &pass_number.to_le_bytes()],
-        constraint = gatekeeper.can_access(&authority, GatekeeperKeyFlags::ISSUE),
+        seeds = [PASS_SEED, subject.as_ref(), network.key().as_ref(), & pass_number.to_le_bytes()],
+        constraint = gatekeeper.can_access(& authority, GatekeeperKeyFlags::ISSUE),
         bump
     )]
-    pub pass: Account<'info, Pass>,
-    pub network: Account<'info, GatekeeperNetwork>,
-    pub gatekeeper: Account<'info, Gatekeeper>,
+    pub pass: Box<Account<'info, Pass>>,
```

Comment:
The Box<> for memory issues

Reply:
Can you be a little more specific on the memory issues? Why exactly is this requirement here and what happens if that is not wrapped in a `Box`?

Reply:
The specific error without wrapping in `Box` is `Program failed to complete: Access violation in stack frame 3 at address 0x200003c28 of size 8 by instruction #32183`